Bottom: 098e8e06a164a5654b0c9bbb194b7647a5cc369d
Top:    a5f9c00e785ef94e8c5890381969b74f6c3ebaf2
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-08-28 02:38:47 +0100

Refresh of work-in-progress-on-subcomposi

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index c12f4b2..3ea1c38 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -966,16 +966,10 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
   glLoadIdentity ();
   glTranslatef (0.0f, 0.0f, -Z_NEAR);
 
-<<<<<<< current
+  glEnable (GL_STENCIL_TEST);
   glClearColor (port->offlimits_color.red / 65535.,
                 port->offlimits_color.green / 65535.,
                 port->offlimits_color.blue / 65535.,
-=======
-  glEnable (GL_STENCIL_TEST);
-  glClearColor (gport->offlimits_color.red / 65535.,
-                gport->offlimits_color.green / 65535.,
-                gport->offlimits_color.blue / 65535.,
->>>>>>> patched
                 1.);
 
   glStencilMask (~0);
